modules/react/common/lib/utils/insights.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="@types/node" />
```

Comment:
@alanbsmith this should fix it. I'm not sure why we need it in some files and we don't need it in others, that is a mystery to me. But this should tell the compiler that we're using node.